sc-machine.ini

```diff
@@ -10,7 +10,7 @@ dump_memory_statistics = false
 dump_memory_statistics_period = 1800
 
 repo_path = ./kb.bin
-extensions_path = ./bin/extensions
+extensions_path = ./build/lib/extensions
```

Comment:
We aren't really sure about the build type being used, this path should be either `./build/Release/lib/extensions` or `./build/Debug/lib/extensions`.

Our options:
- Extensions themselves should go elsewhere (though this would confuse the debug/release builds of extensions and core libs causing all kinds of hazards, not recommended)
- The path to the extensions should be resolved against the sc-memory binary (in contrary to every other path resolve in the config file which will cause surprises, not recommended as well)
- This should be configured to an empty set of paths by default, leaving the consumer to explicitly set the extensions path beforehand. Could work, but:
  - we need to figure out nested configuration so that using config from the underlying repo (not necessarily sc-machine) with option addition/override would be feasible
  - we also need to figure out how to deliver the extensions that are now part of sc-machine repo to the folder that will be set by the user (some kind of installation routine should populate the extension folder with our default extensions on sc-machine installation/update.